BigBaseV2/src/hooking.cpp

```diff
@@ -105,13 +110,21 @@ namespace big
 
 		m_send_net_info_to_lobby.enable();
 
+		m_chat_receive_hook.enable();
+
+		m_censor_chat_text_hook.enable();
+
 		m_enabled = true;
 	}
 
 	void hooking::disable()
 	{
 		m_enabled = false;
 
+		m_censor_chat_text_hook.disable();
+
+		m_chat_receive_hook.disable();
```

Comment:
recieve is incorrect, it should be receive

Reply:
It already is